### DISCUSSION TITLE
Domain setup

### DISCUSSION THREAD

**Discussion by Elizaveta1105** [inbucket/inbucket#240] [Q&A] [answered]
Hello! I've set up Inbucket on my desktop, but I don't receive emails to the domain that I've specified in config.md.
Does anyone know which additional settings I need to do to receive the emails?

**Comment by jhillyerd** [comment, marked as answer]
The answer to this will depend on where the emails are coming from, and how much control you have over that service.

If the system generating the emails lets you set an SMTP host (sometimes called a smart host or relay), set that to your desktops address, port 2500, ie: `192.168.1.100:2500`.   Some services may not let you specify the port, in which case you'll need to configure Inbucket to run SMTP on port 25 instead.

Alternataively, if they emails being sent from your desktop, you can probably get away with setting a hosts entry (https://www.howtogeek.com/howto/27350/beginner-geek-how-to-edit-your-hosts-file/).  If they are coming from another machine, or a VM on your desktop, then you will need to create a DNS MX entry to do this, which varies a lot depending on the domain you are trying to capture.

**Reply by techsachin1** [reply]
hi i have install inbucket on my vm. do i need domain name to configure it.?  or any other way to test inbucket on my local machine
